backend/back/src/workflows/workflows.service.ts

```diff
@@ -261,13 +260,6 @@ export class WorkflowsService {
 
 			if (reactions) {
 				this.logger.log(`Updating ${reactions.length} workflow ${workflowId} reactions...`);
-				this.logger.debug(
-					`Deleted workflow reactions: ${JSON.stringify(
-						await queryRunner.manager.delete(WorkflowArea, {
-							id: In(workflow.reactions.map((reaction) => reaction.id)),
-						}),
-					)}`,
-				);
```

Comment:
why did you remove this ? the reactions are not deleted now

Reply:
Didn't really saw that ahaha, just tought of deleting the debug log